librisworks/src/main/groovy/se/kb/libris/mergeworks/compare/Classification.groovy

```diff
@@ -69,10 +67,59 @@ class Classification extends StuffSet {
         deweyEdition(v1) > deweyEdition(v2) ? v1 : v2
     }
 
-    int deweyEdition(String edition) {
+    static int deweyEdition(String edition) {
         Integer.parseInt((edition ?: "0").replaceAll("[^0-9]", ""))
     }
 
+    static void moveAdditionalDewey(Map mergedWork, Collection<Doc> instanceDocs) {
+        def deweyOnMerged = asList(mergedWork['classification']).findAll { Map c -> isDewey(c) }
+        if (deweyOnMerged.size() > 1) {
+            def allDewey = instanceDocs.collect { it.classification() }
+                    .flatten()
+                    .findAll { Map c -> isDewey(c) }
+
+            def preferredDewey = findPreferredDewey(deweyOnMerged, allDewey)
+
+            deweyOnMerged.remove(preferredDewey)
+
+            mergedWork['classification'] = asList(mergedWork['classification']) - deweyOnMerged
+            mergedWork['additionalClassificationDdc'] = (asList(mergedWork['additionalClassificationDdc']) + deweyOnMerged).unique()
```

Comment:
This is a bit hard to follow. I suggest calling `deweyOnMerged` something else here?

Reply:
See [ea7368e](https://github.com/libris/librisxl/pull/1327/commits/ea7368e0fa7613e3c2e7b4acf555e0d1e317df35)